### ISSUE TITLE
Output a text node within the header image's link so that the link has text.

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1783]
This issue was identified during research on https://github.com/INN/largo/issues/1653 using the AXE browser plugin.

> Fix at least one (1) of these issues:
> 
> - Element does not have text that is visible to screen readers
> - aria-label attribute does not exist or is empty
> - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
> - Element's default semantics were not overridden with role="presentation"
> - Element's default semantics were not overridden with role="none"

With a theme image defined, [`largo_header()`](https://github.com/INN/largo/blob/v0.6.4/inc/header-footer.php#L20-L58) outputs this markup:
```html
<h1 class="visuallyhidden">
	<a itemprop="url" href="https://largo.test/">
		<span class="sitename" itemprop="name">Largo Test</span>
		<span class="tagline" itemprop="description"> - A Largo Test Site</span>
	</a>
</h1>
<a itemprop="url" href="https://largo.test/">
	<img class="header_img" src="https://largo.test/wp-content/uploads/2018/02/largo-test-site-1170x183.png" alt="">
</a>
<meta itemprop="logo" content="https://largo.test/wp-content/uploads/2018/02/image-1-336x337.png">
```

Instead, we should output the header text and image in the same link tag, something like this:

```html
<h1 class="">
	<a itemprop="url" href="https://largo.test/">
		<img class="header_img" src="https://largo.test/wp-content/uploads/2018/02/largo-test-site-1170x183.png" alt="">
		<meta itemprop="logo" content="https://largo.test/wp-content/uploads/2018/02/image-1-336x337.png">
		<span class="visuallyhidden sitename" itemprop="name">Largo Test</span>
		<span class="visuallyhidden tagline" itemprop="description"> - A Largo Test Site</span>
	</a>
</h1>
```

**Comment by MirandaEcho** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Output a text node within the header image's link so the link has text](https://trello.com/c/n4JlKFGk/15-output-a-text-node-within-the-header-images-link-so-the-link-has-text)

**Comment by benlk** [comment]
While here, can we remove the JS requirement for the header logo image by switching to a srcset?
